seeds/_sector/dex/clipper_base_base_trades_seed.csv

```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+base,clipper,1,2024-03-16,0xe89ed138c54bf421a80353d24c3c83ede028e3479269c6ffca7a285bd61ffb5f,43,0x4200000000000000000000000000000000000006,0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb,11992283,20789357653427488,73130000000000000000
```

Comment:
This record is not shown in the query result, please recheck.
(Also there is no evt_index 43 in this tx_hash:)
https://basescan.org/tx/0xe89ed138c54bf421a80353d24c3c83ede028e3479269c6ffca7a285bd61ffb5f#eventlog

Reply:
Thank you, @Hosuke! I have fixed that line.
I'm getting this error now:
```
Failure in test dbt_utils_unique_combination_of_columns_clipper_v1_base_trades_block_date__blockchain__project__version__tx_hash__evt_index (models/clipper/base/clipper_base_schema.yml)
18:33:42    Got 154 results, configured to fail if != 0
18:33:42  
18:33:42    compiled Code at target/compiled/spellbook/models/clipper/base/clipper_base_schema.yml/dbt_utils_unique_combination_o_78b180b2b5b2e5f38d179da3bd01dae0.sql
18:33:42  
18:33:42    See test failures:
  ---------------------------------------------------------------------------------------------------------------------------------------------------------------
  select * from hive.test_schema.git_dunesql_5fb3a0d_dbt_utils_unique_combination_of_columns_clipper_v1_base_trades_block_date__blockchain__project__version__tx_
  ```
 
 Should I remove the file `models/clipper/base/clipper_v1_base_trades.sql` as there is another one under `models/_sector/dex` to fix it?

Reply:
There are dupes in this spells.
https://dune.com/queries/3750838

Reply:
But yes, you can remove files added under `models/clipper/` in this PR.